src/Uno.UI/UI/Xaml/ApplicationActivity.Android.cs

```diff
@@ -117,6 +122,11 @@ private void OnLayoutChanged(Rect statusBar, Rect keyboard, Rect navigationBar)
 
 		protected override void OnCreate(Bundle bundle)
 		{
+			if (Uno.CompositionConfiguration.UseCompositorThread)
+			{
+				Uno.UI.Composition.CompositorThread.Start(this);
```

Comment:
Now that I notice, you may have issues of the activity is re-created for some reason, but the app is not.

Reply:
Good point! I've already some pending updates for this feature, it would create lot of noise to look at it right now ... so I've put it in my backlog (https://github.com/unoplatform/uno/issues/5586) to validate and fix it if needed.